### ISSUE TITLE
Version missing reported after golagci-lint upgrade

### ISSUE THREAD

**Issue by abbas-dependable-naqvi** [golangci/golangci-lint#5874]
### Welcome

- [x] Yes, I'm using a binary release within 2 latest [releases](https://github.com/golangci/golangci-lint/releases). Only such installations are supported.
- [x] Yes, I've searched similar [issues on GitHub](https://github.com/golangci/golangci-lint/issues) and didn't find any.
- [x] Yes, I've read the `typecheck` section of the [FAQ](https://golangci-lint.run/welcome/faq/#why-do-you-have-typecheck-errors).
- [x] Yes, I've tried with the standalone [linter](https://golangci-lint.run/usage/linters/) if available (e.g., gocritic, go vet, etc.).
- [x] I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)

### How did you install golangci-lint?

go install

### Description of the problem

After i upgraded my go version to `v1.23` and toolchain to `go.1.23.5`

I am unable to resolve golangci-lint issues.

Here is the PR https://github.com/mattermost/mattermost-plugin-msteams-meetings/pull/153

### Screenshots

![Image](https://github.com/user-attachments/assets/ac111697-f22c-47e1-83cb-841266e39097)

![Image](https://github.com/user-attachments/assets/aa4bd3e0-2cef-409d-afca-ee21019da7bf)

<details>

```console
$ golangci-lint --version
# Paste output here
```

</details>

### Configuration

<details>

```console
# paste configuration file or CLI flags here
```

</details>

### Go environment

<details>

```console
$ go version && go env
# paste output here
```

</details>

### Verbose output of running

<details>

```console
$ golangci-lint cache clean
$ golangci-lint run -v
# paste output here
```

</details>

### A minimal reproducible example or link to a public repository

<details>

```go
// add your code here
```

</details>

### Validation

- [x] Yes, I've included all information above (version, config, etc.).

### Supporter

- [ ] I am a sponsor/backer through [GitHub](https://github.com/sponsors/golangci/) or [OpenCollective](https://opencollective.com/golangci-lint)

**Comment by ldez** [comment]
Hello,

This is because you are using golangci-lint v2 with a golangci-lint v1 configuration.

It's the same thing as the other plugin https://github.com/mattermost/mattermost-plugin-confluence/pull/140

I will open a PR on your repository.

**Comment by abbas-dependable-naqvi** [comment]
Thanks, my issue was resolved swiftly
